python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -38,7 +38,6 @@
     set_input,
     append_input_output,
     append_loss_ops, )
-from paddle.fluid import unique_name
```

Comment:
I saw CI failed a lot, is it due to you deleted this line?

Reply:
Fixed.